### PR TITLE
Release service auth 1.0

### PR DESCRIPTION
**Pull request by ben-polinsky**
- Bump dependencies
- Add usage to readme

part of #102 
fixes #661

### CODE REVIEW

File: packages/service/package.json [hunk 1]
```diff
@@ -30,10 +30,10 @@
     "openid-client": "^4.7.4"
   },
   "devDependencies": {
-    "@itwin/build-tools": "^3.3.0",
-    "@itwin/core-bentley": "^3.3.0",
-    "@itwin/core-common": "^3.3.0",
-    "@itwin/eslint-plugin": "^3.3.0",
+    "@itwin/build-tools": "^4.0.0-dev.52",
```
Comment (aruniverse) [inline packages/service/package.json:33]: maybe we go to 1.0 with dev deps using ^3.3
and then update to 4.0 once its released;

main thing was updating peer

**Review by aruniverse — APPROVED** [review]
lgtm; @paulius-valiunas any thoughts?

**Comment by paulius-valiunas** [comment]
@aruniverse sorry I missed your comment initially... I would like to complete #151 first and perhaps convert these packages to ES Modules

**Review by aruniverse — APPROVED** [review]

File: packages/service/package.json [hunk 2]
```diff
@@ -49,7 +49,7 @@
     "rimraf": "^3.0.2",
     "sinon": "^15.0.1",
     "source-map-support": "^0.5.9",
-    "typescript": "~4.3.5"
+    "typescript": "~5.0.2"
   },
   "peerDependencies": {
     "@itwin/core-bentley": ">=3.3.0"
```
Comment (aruniverse) [inline packages/service/package.json:55]:
```suggestion
    "@itwin/core-bentley": "^3.3.0 || ^4.0.0"
```

Comment (ben-polinsky) [reply packages/service/package.json:55]: Right thanks.

**Pull request merged**